orangecontrib/timeseries/widgets/owperiodogram.py

```diff
@@ -42,15 +42,18 @@ def replot(self):
             return
 
         palette = self.get_palette()
-        for i, attr in enumerate(self.selection):
+        datae = [self.periodogram(attr) for attr in self.selection]
```

Comment:
As a former Latin student I must comment. :) Data is already plural of the noun datum.

Reply:
κῦδος. :)